opm/autodiff/RedistributeDataHandles.hpp

```diff
@@ -34,7 +34,7 @@ namespace Opm
 
 template <class Grid>
 inline void distributeGridAndData( Grid& ,
-                                   Opm::EclipseStateConstPtr eclipseState,
+                                   Opm::EclipseStateConstPtr ,
```

Comment:
Cool.  Do you think you could remove the namespace qualification too?  We are, after all, already in namespace `Opm`.